lib/ansible/modules/cloud/vmware/vmware_host_facts.py

```diff
@@ -277,6 +279,8 @@ def get_system_facts(self):
             'ansible_product_serial': sn,
             'ansible_bios_date': self.host.hardware.biosInfo.releaseDate,
             'ansible_bios_version': self.host.hardware.biosInfo.biosVersion,
+            'ansible_uptime': self.host.summary.quickStats.uptime,
+            'ansible_in_maintenance_mode': self.host.runtime.inMaintenanceMode,
```

Comment:
Change looks good. Could you please add an integration test for this change here - https://github.com/ansible/ansible/blob/064cd63f3d1c70019ca10a5eba75212ee9705c16/test/integration/targets/vmware_host_facts/tasks/main.yml#L34